modules/developer-cli-odo-installing-odo-on-linux-rpm.adoc

```diff
@@ -0,0 +1,56 @@
+[id="installing-odo-on-linux-rpm_{context}"]
+
+= Installing {odo-title} on {op-system-base-full} using an RPM 
+
+For {op-system-base-full}, you can install the `{odo-title}` CLI as an RPM.
+
+.Procedure
+
+. Register with Red Hat Subscription Manager:
```

Comment:
Consider giving us more of a reason as to why we'd want to install the odo cli as RPM. Something simple like this formula: 
`To <state benefit/why user would care> on Red Hat Enterprise Linux (RHEL), install the odo CLI as a RPM.` 

I'm not sure of the reason, but maybe the benefit is something like this:
`To work with simple binaries in complex operating systems on RHEL, install the odo CLI as a RPM.`

Reply:
As before, I am following the conventions in oc docs, and they do not explain about RPMs, as it is an assumed knowledge.